astropy/cosmology/io/mapping.py

```diff
@@ -135,6 +162,8 @@ def to_mapping(
         Whether to add the Cosmology's metadata as an item to the mapping (if
         `False`, default) or to merge with the rest of the mapping, preferring
         the original values (if `True`)
+    rename : dict or None, optional, keyword-only
+        A dictionary mapping keys in the Cosmology class' to keys.
```

Comment:
Here, the description is a bit unclear: "keys of the map"? Generally, is "keys in the Cosmology class" the right term? Would "attributes" be better?

Reply:
Agreed. Let me clean that up.

Reply:
I changed it to "fields", which is the parlance of Python dataclasses.

Reply:
That sounds better. I was wondering about `parameter` since this is about models; but mentioning this just in case you think it is better.